spec.md

```diff
@@ -2073,6 +2073,9 @@ message NodeStageVolumeRequest {
   // CO SHALL be responsible for creating the directory if it does not
   // exist.
   // This is a REQUIRED field.
+  // This field overrides the general CSI size limit.
+  // The size of this field CAN vary depend on what the underlying OS
+  // supports. The minial support size SHOULD be at least 128 bytes.
   string staging_target_path = 3;
 
```

Comment:
Consider adding a new error code, maybe `OUT_OF_RANGE`, that SP can return if it does not support the given length of `target_path`.

Reply:
Can we use the https://github.com/container-storage-interface/spec/blob/master/spec.md#error-scheme invalid argument error code? I dont know if it is worth it to add a special error code for this case.

Reply:
Ya, I guess this would fall under the existing `INVALID_ARGUMENT`